recipe/meta.yaml

```
@@ -45,13 +43,8 @@ requirements:
 
 test:
   commands:
-    {% set lib_idents = [ "Xdmcp" ] %}
-    {% for lib_ident in lib_idents %}
-    - test -f $PREFIX/lib/lib{{ lib_ident }}.dylib  # [osx]
-    - test -f $PREFIX/lib/lib{{ lib_ident }}.so  # [linux]
-    - if not exist %PREFIX%/Library/lib/lib{{ lib_ident }}.dll.a exit /b 1  # [win]
-    - if not exist %PREFIX%/Library/bin/msys-{{ lib_ident }}-*.dll exit /b 1  # [win]
-    {% endfor %}
+    - test -f $PREFIX/lib/libXdmcp.dylib  # [osx]
+    - test -f $PREFIX/lib/libXdmcp.so  # [linux]
```

```suggestion
    - test -f $PREFIX/lib/libXdmcp.dylib  # [osx]
    - test -f $PREFIX/lib/libXdmcp.so  # [linux]
    - if not exist %LIBRARY_LIB%/libXdmcp.a exit /b 1  # [win]
    - if not exist %LIBRARY_LIB%/libXdmcp.la exit /b 2  # [win]
```
Do we want these Windows libs converted over to `.lib` files? Something like [this](https://github.com/AnacondaRecipes/mpfr-feedstock/blob/be9ceaa2a5b2e67b417cd8a7b3b6c4349438d554/recipe/bld.bat#L10).

---

I don't know if we should do it that way. 
conda-forge builds on win-64 these files:
```
Library/bin/msys-Xdmcp-6.dll
Library/include/X11/Xdmcp.h
Library/lib/libXdmcp.dll.a
Library/lib/pkgconfig/xdmcp.pc
```

see https://conda-metadata-app.streamlit.app/?q=conda-forge%2Fwin-64%2Fxorg-libxdmcp-1.1.5-h0e40799_0.conda&with_broken=true

I'll test them here. 

Which downstream packages require or can these converted filenames? I don't see any https://conda-metadata-app.streamlit.app/Search_by_file_path?path=Library%2Flib%2FlibXdmcp.dll.a

---

I had the conversions backwards, it seems. Looks like `.dll.a` is a valid extension on Windows for MinGW or MSYS2 environments. In fact, that link I provided has us creating a `.dll.a` from a `.lib` for compatibility.

---

I dropped Windows.